src/App.vue

```diff
@@ -106,14 +125,16 @@
             no-flags
             no-use-browser-locale
             required
+            :loader="hasLoaderActive"
+            :error="hasErrorActive"
             @update="onUpdate2"
           />
         </div>
         <br>
         <div class="component">
           <h3>Datas returned with "onUpdate" event</h3>
           <hr>
-          <b>v-model</b> : {{ phoneNumber }}
+          <b>v-model</b> : {{ phoneNumber }}agirdhar
```

Comment:
Sneaky, leaving your tag in random places? ;)

Reply:
I don't know, how my name is all around. Let me fix this